src/App.js

```diff
@@ -1,12 +1,22 @@
-import React, { Component } from 'react';
+import React from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export class App extends Component {
-  state = {};
+export class App extends React.Component {
+  state = {
+    query: '',
+  };
+
+  handleQuery = (event) => {
+    this.setState({ query: event.target.value });
+  }
 
   render() {
+    const selectedMovies = moviesFromServer
+      .filter(movie => movie.title.toLowerCase()
+        .startsWith(this.state.query.toLowerCase()));
+
```

Comment:
```suggestion
    const selectedMovies = moviesFromServer
      .filter(movie => movie.title.toLowerCase()
        .startsWith(this.state.query.toLowerCase()));

```
Your code filter only by ```title```. Reread instruction.

Reply:
well, there is some ambiguity in the instruction, but i think you are right